l10n_it_dichiarazione_intento/data/sequence.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <record id="dichiarazione_intento_seq" model="ir.sequence">
+            <field name="name">Dichiarazione Intento sequence</field>
```

Comment:
Qui e in tutte le altre strighe simili  in cui compaiono, le forme 
`Dichiarazione di intento`, `Dichiarazione intento` e `Dichiarazione d'intento` andrebbero tradotte come "Declaration of Intent".

Vedi
http://www.vatassociation.org/vat-news/news-from-iva-members/76-vat-in-italy-law-reform-act
https://support.microsoft.com/it-it/help/4021866/italy-2017-updates-for-declaration-of-intent-for-microsoft-dynamics-na